### DISCUSSION TITLE
URL Link

### DISCUSSION THREAD

**Discussion by twilson9034** [Budibase/budibase#3419] [Help] [answered]
I am having an issue with a url link.

The nav bar works just fine with the same link. when I use the link in the URL for anything on the screen it takes me to the wrong screen.

link is /user/info 
 other then the nav bar the link take me to /media-links

**Comment by twilson9034** [comment]
helps to add the "s" to user.

**Comment by mjashanks** [comment]
I saw you marked this as "Unanswered" again, so I tried it. It works ok for me. I have a screen with URL `/test`, and a link with the URL `/test` - it directed me ok.

You might find that you need to refresh your app, after making the change with the `s`.

**Comment by twilson9034** [comment, marked as answer]
This is what i found out, not sure if its suppose to be that way or not.

When going to another screen I needed to pass the user id and was not using the param syntax correct. I was going to /users/{{user_.id}}/info , which seemed correct but the route was not set as users/:id/info so the user_.id had no place to go.

Once I got the syntax correct for sending Parameters into the url I still had an issue.  I had copied the user/;id screen so I thought it all would work but for some reason under the data provider -> define filter 
![image](https://user-images.githubusercontent.com/94488623/142447471-e5bc893f-6f88-4791-89ff-6e9f01763126.png)
the equals binding or value needs to be set to value. 

Not sure why on the user/;id screen its set to binding and the other screens need it set to value.
